tests/notifications-api.test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var request = require('supertest');
+var should = require('should');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['should' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/768301334/issues/source?bid=2452379&fileBranchId=2457009#l4)